MantidPlot/src/Mantid/MantidUI.cpp

```diff
@@ -1377,6 +1361,15 @@ Table *MantidUI::createDetectorTable(
       }
       const QString isMonitorDisplay = isMonitor ? "yes" : "no";
       colValues << QVariant(specNo) << QVariant(detIds);
+      if (isScanning) {
+        std::set<int> timeIndexSet;
+        for (auto def : spectrumInfo.spectrumDefinition(wsIndex)) {
+          timeIndexSet.insert(int(def.second));
+        }
+
+        QString timeIndexes = createTruncatedList(timeIndexSet);
+        colValues << QVariant(timeIndexes);
+      }
```

Comment:
Just to be clear, are you saying that you could have a spectrum with 2 detectors, one contributing time indexes [1, 2] and another contributing time indexes [2, 3]. You would then see the time indexes as [1, 2, 3] without knowing which detectors had which time indexes?

I would think we can ignore that, as it should be easy to see when you have multiple time indexes and multiple detectors. The detector table does not always give the complete information in any case, truncating long lists.

The other alternative would be to do something like "1 (1, 2), "2 (2, 3)" in the detector part of the table. I think that would get more difficult to read clearly though.